code/modules/admin/verbs/randomverbs.dm

```diff
@@ -472,6 +472,30 @@ Traitors and the like can also be revived with the previous role mostly intact.
 	admin_ticket_log(M, msg)
 	SSblackbox.add_details("admin_verb","Rejuvinate") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
 
+/client/proc/cmd_admin_freeze(mob/living/M in GLOB.mob_list)
+	set category = "Special Verbs"
+	set name = "Freeze Player"
+	if(!holder)
+		to_chat(src, "Only administrators may use this command.")
+		return
+	if(!istype(M))
+		return
+
+	if(!M.adminfrozen)
+		var/sleepies = M.AmountSleeping()
+		M.adminfrozen = sleepies ? sleepies : 1
+		M.SetSleeping(20000)//2k seconds to get your admin shit together
+		M.adminfreezeoverlay = new()
+		M.add_overlay(M.adminfreezeoverlay)
+		log_admin("[key_name(usr)] froze [key_name(M)]!")
+		to_chat(M, "<span class='userdanger'>You have been frozen by Administrator [usr.key]!</span>")

```

Comment:
Maybe increase the size yo

Reply:
Doesn't get bigger than this without becoming unreadable. `userdanger` is what spams so hard when you're attacked. As high as a quarter of your chat window. Even worse on smaller screens.

Reply:
oh right, cool